pkg/cmd/codespace/codespace_selector.go

```diff
@@ -25,6 +26,7 @@ func AddCodespaceSelector(cmd *cobra.Command, api apiClient) *CodespaceSelector
 
 	cmd.PersistentFlags().StringVarP(&cs.codespaceName, "codespace", "c", "", "Name of the codespace")
 	cmd.PersistentFlags().StringVarP(&cs.repoName, "repo", "R", "", "Filter codespace selection by repository name (user/repo)")
+	cmd.Flags().StringVar(&cs.repoOwner, "repo-owner", "", "To filter by the `username` of the repository owner")
```

Comment:
Can you use `PersistentFlags` here? The `ports` command uses some sub-commands and `PersistentFlags` ensures that these flags flow to them too (might be worth adding a comment explaining that too)

Reply:
Sorry my bad, fixed now